src/state_point.F90

```diff
@@ -236,10 +236,10 @@ subroutine write_state_point()
           ! Get pointer to tally
           tally => tallies(i)
 
-          call sp % write_data(len(tally % label), "label_size", &
+          call sp % write_data(len(tally % name), "name_size", &
                group="tallies/tally " // trim(to_str(tally % id)))
-          if (len(tally % label) > 0) then
-            call sp % write_data(tally % label, "label", &
+          if (len(tally % name) > 0) then
+            call sp % write_data(tally % name, "name", &
```

Comment:
Maybe we should move the tally names to the summary file as well so that everything is consistent. Do you see any issues with that?